contracts/NFTVault.sol

```diff
@@ -56,6 +54,16 @@ contract NFTVault is IERC721Receiver, IERC1155Receiver, AccessControl {
         return keys.balanceOf(msg.sender);
     }
 
+    function getTokenType(address addr)
```

Comment:
Add comments and check naming style and formating of declared functions params. They are not unified through the file - compare declaration of function above with `backDoorERC721`  for example